creator/Documents/ScoreboardIntroduction.md

```diff
@@ -82,62 +82,62 @@ This command returns a list of all of the objectives in the world.
 To remove an objective from the game use:
 
 ```
-/scoreboard objective remove [objective name]
+/scoreboard objective remove <objective: string>
```

Comment:
I made various changes throughout the article to the command syntax to more closely match how it appears in game. Required arguments are listed with angled brackets `<>`, while optional arguments are listed with square brackets `[]`. Additionally, literal values are printed as-is while non-literal values have their type listed after a colon.
There is one exception to this I will point out down below, on line 160.